media-video/mplayer/mplayer-1.4.ebuild

```diff
@@ -0,0 +1,626 @@
+	fi
+}
+
+src_unpack() {
+	if [[ ${PV} = *9999* ]]; then
+		subversion_src_unpack
+		cd "${WORKDIR}"
+		rm -rf "${WORKDIR}/${P}/ffmpeg/"
+		( S="${WORKDIR}/${P}/ffmpeg/" git-2_src_unpack )
+	else
+		unpack ${A}
+	fi
+
+	if [[ ${PV} = *9999* ]] || [[ "${PV%_rc*}" = "${PV}" ]]; then
+		cd "${S}"
+		cp "${FILESDIR}/dump_ffmpeg.sh" . || die
+		chmod +x dump_ffmpeg.sh
+		./dump_ffmpeg.sh || die
+	fi
+
+	if ! use truetype; then
+		unpack font-arial-iso-8859-1.tar.bz2 \
+			font-arial-iso-8859-2.tar.bz2 \
+			font-arial-cp1250.tar.bz2
+	fi
+}
+
+src_prepare() {
+	default
+
+	local svf=snapshot_version
+	if [[ ${PV} = *9999* ]]; then
+		# Set SVN version manually
+		subversion_wc_info
+		printf "${ESVN_WC_REVISION}" > $svf
+	else
+		has_version '>media-video/ffmpeg-3.5'
+	fi
+	if [ ! -f VERSION ] ; then
+		[ -f "$svf" ] || die "Missing ${svf}. Did you generate your snapshot with prepare_mplayer.sh?"
+		local sv=$(<$svf)
+		printf "SVN-r${sv} (Gentoo)" > VERSION
+	fi
+
+	# fix path to bash executable in configure scripts
+	sed -i -e "1c\#!${EPREFIX}/bin/bash" configure version.sh || die
+
+	# Use sane default for >=virtual/udev-197
+	sed -i -e '/default_dvd_device/s:/dev/dvd:/dev/cdrom:' configure || die
+}
+
+src_configure() {
+	local myconf=""
+	local uses i
+
+	# set LINGUAS
+	[[ -n $LINGUAS ]] && LINGUAS="${LINGUAS/da/dk}"
+	[[ -n $LINGUAS ]] && LINGUAS="${LINGUAS/zh/zh_CN}" #482968
+
+	# mplayer ebuild uses "use foo || --disable-foo" to forcibly disable
+	# compilation in almost every situation. The reason for this is
+	# because if --enable is used, it will force the build of that option,
+	# regardless of whether the dependency is available or not.
+
+	###################
+	#Optional features#
+	###################
+	# disable svga since we don't want it
+	# disable arts since we don't have kde3
+	# always disable internal ass
+	# disable opus and ilbc since it only controls support in internal
+	#         ffmpeg which we do not use
+	myconf+="
+		--disable-svga --disable-svgalib_helper
+		--disable-ass-internal
+		--disable-arts
+		--disable-directfb
+		--disable-kai
+		--disable-libopus
+		--disable-libilbc
+		--disable-xvmc
+		$(use_enable network networking)
+		$(use_enable joystick)
+	"
+	uses="bl bluray enca ftp rtc vcd" # nemesi <- not working with in-tree ebuild
+	myconf+=" --disable-nemesi" # nemesi automagic disable
+	for i in ${uses}; do
+		use ${i} || myconf+=" --disable-${i}"
+	done
+	use bidi  || myconf+=" --disable-fribidi"
+	use ipv6  || myconf+=" --disable-inet6"
+	use libass || myconf+=" --disable-ass"
+	use nut   || myconf+=" --disable-libnut"
+	use rar   || myconf+=" --disable-unrarexec"
+	use samba || myconf+=" --disable-smb"
+	use lirc  || myconf+=" --disable-lirc --disable-lircc --disable-apple-ir"
+
+	# enable openssl support for https
+	# the binary cannot be redistributed if set
+	# by default gnutls will be enabled
+	if use ssl ; then
+		myconf+=" $(usex !gnutls '--disable-gnutls --enable-openssl-nondistributable')"
```

Comment:
I gave you a bad example. This will break if you have the `gnutls` USE flag enabled. Please update this line to:
```
myconf+=" $(usex !gnutls '--disable-gnutls --enable-openssl-nondistributable' '')"
```